web/src/message_view.ts

```diff
@@ -116,7 +125,22 @@ export function update_hash_to_match_filter(filter, trigger) {
     }
 }
 
-function create_and_update_message_list(filter, id_info, opts) {
+type IdInfo = {
```

Comment:
This type name is probably too vague. `TargetMessageIdInfo` might be appropriate?

Reply:
Renamed in a followup commit/PR